easybuild/easyconfigs/i/IPython/IPython-5.8.0-fosscuda-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,227 @@
+        'checksums': ['b7493f73a2febe0dc33d51c99b474547f7f6c0b2c8fb2b21f453eef204c12148'],
+    }),
+    ('testpath', '0.4.2', {
+        'source_urls': ['https://pypi.python.org/packages/source/t/testpath/'],
+        'checksums': ['b694b3d9288dbd81685c5d2e7140b81365d46c29f5db4bc659de5aa6b98780f8'],
+    }),
+    ('nose', '1.3.7', {
+        'source_urls': ['https://pypi.python.org/packages/source/n/nose/'],
+        'checksums': ['f1bffef9cbc82628f6e7d7b40d7e255aefaa1adb6a1b1d26c69a8b79e6208a98'],
+    }),
+    ('configparser', '3.5.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/c/configparser/'],
+        'use_pip': True,
+        'checksums': ['5308b47021bc2340965c371f0f058cc6971a04502638d4244225c49d80db273a'],
+    }),
+    ('defusedxml', '0.5.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/d/defusedxml/'],
+        'checksums': ['24d7f2f94f7f3cb6061acb215685e5125fbcdc40a857eff9de22518820b0a4f4'],
+    }),
+    ('nbconvert', '5.4.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/n/nbconvert/'],
+        'use_pip': True,
+        'checksums': ['a8a2749f972592aa9250db975304af6b7337f32337e523a2c995cc9e12c07807'],
+    }),
+    ('nbformat', '4.4.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/n/nbformat/'],
+        'use_pip': True,
+        'checksums': ['f7494ef0df60766b7cabe0a3651556345a963b74dbc16bc7c18479041170d402'],
+    }),
+    ('MarkupSafe', '1.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/m/MarkupSafe/'],
+        'checksums': ['a6be69091dac236ea9c6bc7d012beab42010fa914c459791d627dad4910eb665'],
+    }),
+    ('Jinja2', '2.10', {
+        'source_urls': ['https://pypi.python.org/packages/source/j/Jinja2/'],
+        'checksums': ['f84be1bb0040caca4cea721fcbbbbd61f9be9464ca236387158b0feea01914a4'],
+    }),
+    ('Send2Trash', '1.5.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/s/Send2Trash/'],
+        'checksums': ['60001cc07d707fe247c94f74ca6ac0d3255aabcb930529690897ca2a39db28b2'],
+    }),
+    ('webencodings', '0.5.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/w/webencodings/'],
+        'checksums': ['b36a1c245f2d304965eb4e0a82848379241dc04b865afcc4aab16748587e1923'],
+    }),
+    ('html5lib', '1.0.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/h/html5lib/'],
+        'checksums': ['66cb0dcfdbbc4f9c3ba1a63fdb511ffdbd4f513b2b6d81b80cd26ce6b3fb3736'],
+    }),
+    ('bleach', '3.0.2', {
+        'source_urls': ['https://pypi.python.org/packages/source/b/bleach/'],
+        'checksums': ['48d39675b80a75f6d1c3bdbffec791cf0bbbab665cf01e20da701c77de278718'],
+    }),
+    ('functools32', '3.2.3-2', {
+        'source_urls': ['https://pypi.python.org/packages/source/f/functools32/'],
+        'checksums': ['f6253dfbe0538ad2e387bd8fdfd9293c925d63553f5813c4e587745416501e6d'],
+    }),
+    ('jsonschema', '2.6.0', {
```

Comment:
on a node without access to PYPi, but after running `--fetch` from a node with access, installation of `jsonschema` is trying to fetch `vcversioner`, so it should be added before this line.

Again (e.g. #7696), I don't know why `exts_download_dep_fail = True` is not picking up on this...